src/styles/styles.css

```diff
@@ -172,6 +172,10 @@
         font-size: 1.5rem; /* 24px */
         line-height: 1.75rem; /* 28px */
     }
+    .buttons-container .buttono {
```

Comment:
❓ is this normal in css to have space like `.buttons-container .buttono` instead of `.buttons-container.buttono` 😕

Reply:
No space means both classes, an element that has both.
A space means buttono element that is descendent of buttons-container.

Reply:
I probably did a mistake will check it tomorrow